package.json

```diff
@@ -1,81 +1,68 @@
-        "@tiptap/extension-bullet-list": "^2.1.10",
-        "@tiptap/extension-character-count": "^2.1.10",
-        "@tiptap/extension-document": "^2.1.10",
-        "@tiptap/extension-history": "^2.0.3",
-        "@tiptap/extension-italic": "^2.1.10",
-        "@tiptap/extension-list-item": "^2.1.10",
-        "@tiptap/extension-paragraph": "^2.1.10",
-        "@tiptap/extension-placeholder": "^2.1.10",
-        "@tiptap/extension-text": "^2.1.10",
-        "@tiptap/pm": "^2.0.3",
-        "@tiptap/vue-2": "^2.1.10",
-        "@vitejs/plugin-vue2": "^2.2.0",
-        "@vitest/coverage-v8": "^0.32.2",
-        "@vue/babel-preset-app": "^3.7.0",
-        "@vue/test-utils": "^1.3.6",
-        "babel-loader": "^8.0.5",
-        "babel-polyfill": "^6.26.0",
-        "current-script-polyfill": "^1.0.0",
-        "eslint": "^5.15.3",
-        "flush-promises": "^1.0.2",
-        "happy-dom": "^9.20.3",
-        "lodash.get": "^4.4.2",
-        "lodash.unescape": "^4.0.1",
-        "prettier": "2.8.8",
-        "sass": "1.60.0",
-        "tippy.js": "^4.3.1",
-        "vite": "^4.3.9",
-        "vitest": "^0.32.2",
-        "vue": "^2.6.10",
-        "webpack": "^4.31.0"
-    },
-    "browserslist": [
-        "> 1%",
-        "last 2 versions",
-        "ie >= 9"
-    ],
-    "dependencies": {
-        "vite-plugin-css-injected-by-js": "^3.3.0",
-        "vue-template-compiler": "^2.7.14"
-    }
+	"name": "@mirus/tiptap-editor",
+	"version": "2.1.9",
+	"description": "custom setup for the tiptap editor",
+	"repository": "https://github.com/mirusresearch/tiptap-editor",
+	"type": "module",
+	"bugs": {
+		"url": "https://github.com/mirusresearch/tiptap-editor"
+	},
+	"homepage": "https://github.com/mirusresearch/tiptap-editor",
+	"author": "alec@mirusresearch.com",
+	"license": "MIT",
+	"main": "dist/tiptap-editor.js",
+	"browser": {
+		"./sfc": "src/tiptap-editor.vue"
+	},
+	"files": ["dist/*", "src/*", "public/*", "*.json", "*.js"],
+	"exports": {
+		".": {
+			"import": "./dist/tiptap-editor.js",
+			"require": "./dist/tiptap-editor.umd.cjs"
+		}
+	},
+	"scripts": {
+		"serve": "vite",
+		"build": "vite build",
+		"preview": "vite preview",
+		"format": "prettier --write \"src\" \"test\"",
+		"validate:format": "prettier --check \"src\" \"test\"",
+		"test": "vitest run",
+		"test:watch": "vitest",
+		"coverage": "vitest run --coverage"
+	},
+	"devDependencies": {
+		"@tiptap/core": "2.4.0",
+		"@tiptap/extension-bold": "2.4.0",
+		"@tiptap/extension-bullet-list": "2.4.0",
+		"@tiptap/extension-character-count": "2.4.0",
+		"@tiptap/extension-document": "2.4.0",
+		"@tiptap/extension-history": "^2.0.3",
+		"@tiptap/extension-italic": "2.4.0",
+		"@tiptap/extension-list-item": "2.4.0",
+		"@tiptap/extension-paragraph": "2.4.0",
+		"@tiptap/extension-placeholder": "2.4.0",
+		"@tiptap/extension-text": "2.4.0",
+		"@tiptap/pm": "2.4.0",
+		"@tiptap/vue-3": "2.4.0",
+		"@vitest/coverage-v8": "^0.32.2",
+		"@vue/compat": "^3.1.0",
```

Comment:
Is the compat build still needed?

Reply:
good question. I dont think so, but I'll double check

Reply:
ya @vue/compat is specifically for vue2. Away it goes!